lisa/tests/scheduler/load_tracking.py

```diff
@@ -973,7 +998,20 @@ def get_trace_cpu_util(self):
             phase_df = df_window(df, (start, end), method='pre', clip_window=True)
 
             for cpu in self.cpus:
-                util = phase_df[phase_df['cpu'] == cpu]['util']
+
+                cpu_phase_df = phase_df[phase_df['cpu'] == cpu].copy()
+
+                df_state_clipped = pd.concat(df_window(df_state[task], (start, end), method='pre', clip_window=True)
+                                             for task, wlgen_task in self.rtapp_profile.items()
+                                             if cpu in wlgen_task.phases[phase-1].cpus)
```

Comment:
Why are we inspecting the CPUs of the previous phase ?

Reply:
phase = 1 and phase = 2 are the two testphases which correspond to wlgen_task.phases[0] and wlgen_task.phases[1].

phase = 0 is the buffer phase which is visible in testdata on some boards (sd845 ?)

I added this:

# TODO: remove that once we have named phases to skip the buffer phase.
if phase == 0:
    continue

to skip over the buffer phase.

Reply:
Right, we need to re-align the number we get from rt-app to look it up inside the profile indeed